lib/phoenix/code_reloader/server.ex

```diff
@@ -78,10 +80,27 @@ defmodule Phoenix.CodeReloader.Server do
   end
 
   defp os_symlink({:win32, _}),
-    do: " On Windows, such can be done by starting the shell with \"Run as Administrator\"."
+    do: " On Windows, you can started the shell with \"Run as Administrator\" at least once time, " <> 
+        "to allow phoenix create symlinks and resolve this problem."
   defp os_symlink(_),
     do: ""
 
+  defp can_symlink() do
+    build_path = Mix.Project.build_path()
+    symlink = Path.join(Path.dirname(build_path), "__phoenix__")
+
+    case File.ln_s(build_path, symlink) do
```

Comment:
Isn't this a recursive symlink? I don't think there is anything wrong with it but it would probably be best to avoid it. Maybe we should try to symlink app_path/priv to priv, which is the symlink we would build at the end of the day.

Reply:
Ignore me, we are using the same link as before. :)

Reply:
Yes, and I think this is not the best way too. Those codes that create a temp symlink and then remove them maybe just a opportunistic way.
What the `can_symlink` function actually need to do is just, find that is the task are running 
 with `Administrator` and have permission to create symlinks.
But since those codes exist in the old version and the have works well, I became lazy to find other solutions, just move them into the `can_symlink` function.